Bottom: 3b6f87337be02c422c40f75cd3b028d61cd15c30
Top:    37e8def6c233c6c25cdeba8c0a680b37eed13204
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:29:28 -0500

Refresh of fixup-tmp-rework-ip-address-network

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index be3f73ab..e9587369 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -50,6 +50,18 @@
     - bacula_restores
   - name: Configure Docker networking
     role: docker-networking
+    net_connection: '{{ daedalus_net_connection }}'
+    subnet: '{{ daedalus_subnet }}'
+    services:
+    - svn
+    - openldap
+    - exim4
+    - gitlab
+    - wiki
+    - mantisbt
+    - build
+    - testssh
+    - buildbot
     tags:
     - docker_networking
   - name: Build the openssl Docker volume
